controller_manager:
  ros__parameters:
    update_rate: 100 # Hz
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    rocket_kaya_controller:
      type: rocket_kaya_controller/RocketKayaController

rocket_kaya_controller:
  ros__parameters:
    odom_frame_id: odom
    base_frame_id: base_link
    joints: ['wheel_1_joint', 'wheel_2_joint', 'wheel_3_joint']
    kinematics:
      r: 0.041275
      d: 0.116720
      alpha: [ 1.047197551196598, 3.141592653589793, -1.047197551196598 ]
    cmd_vel_timeout: .5
    pose_covariance_diagonal : [0.001, 0.001, 0.001, 0.001, 0.001, 0.01]
    twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.01]
    linear_vel_limit: .NaN
    linear_accel_limit: .NaN
    angular_vel_limit: .NaN
    angular_accel_limit: .NaN
    